### PR TITLE
[bug](compile) fix vscan node compile error

### PR DESCRIPTION
**Pull request by ByteYue**
# Proposed changes

Issue Number: close #xxx

## Problem summary

conflict merge of #15604 and #15618

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [x] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [x] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [x] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by adonis0147 — DISMISSED** [review]
LGTM

File: be/src/vec/exec/scan/vmeta_scan_node.cpp [hunk 1]
```diff
@@ -53,7 +53,7 @@ Status VMetaScanNode::_init_scanners(std::list<VScanner*>* scanners) {
     }
     for (auto& scan_range : _scan_ranges) {
         VMetaScanner* scanner =
-                new VMetaScanner(_state, this, _tuple_id, scan_range, _limit_per_scanner);
+                new VMetaScanner(_state, this, _tuple_id, scan_range, _limit_per_scanner, _state->runtime_profile());
```
Comment (morningman) [inline be/src/vec/exec/scan/vmeta_scan_node.cpp:56]:
```suggestion
                new VMetaScanner(_state, this, _tuple_id, scan_range, _limit_per_scanner, runtime_profile());
```

**Review by morningman — DISMISSED** [review]
LGTM

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 37.19 seconds
 load time: 526 seconds
 storage size: 17123347030 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230111044323_clickbench_pr_77341.html

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**